src/sdk/pynni/tests/test_builtin_tuners.py

```diff
@@ -1,369 +0,0 @@
-# Copyright (c) Microsoft Corporation.
```

Comment:
why delete this file?

Reply:
I was trying to revert the changes and I deleted the file by mistake. I will fix it today.